src/assets/thirdPartyServices/snet/zeta36_chess_alpha_zero/index.js

```diff
@@ -1,31 +1,36 @@
 import React from "react";
 import { hasOwnDefinedProperty } from "../../../../utility/JSHelper";
 import Button from "@material-ui/core/Button";
+import { Zeta6AlphaZero } from "./alpha_zero_pb_service";
+
+
+const initialUserInput = {
+  uid: "",
+  move: "",
+  cmd: "",
+};
+
 
 export default class Zeta36ChessAlphaZero extends React.Component {
   constructor(props) {
     super(props);
     this.submitAction = this.submitAction.bind(this);
-    this.handleServiceName = this.handleServiceName.bind(this);
+    //this.handleServiceName = this.handleServiceName.bind(this);
     this.handleFormUpdate = this.handleFormUpdate.bind(this);
-    this.getServiceMethods = this.getServiceMethods.bind(this);
+    //this.getServiceMethods = this.getServiceMethods.bind(this);
```

Comment:
Commented code can be deleted from the file.

Reply:
I just kept them for reference during the test. Once all the test is done we can delete the commented code.